wled00/data/settings_wifi.htm

```diff
@@ -6,18 +6,100 @@
 	<meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport"/>
 	<title>WiFi Settings</title>
 	<script>
-		var d=document;
-		var loc = false, locip;
+		let d = document;
```

Comment:
`var` vs. `let` : I tend to use `var` for globals like `d` and use `let` within functions, so I recommend sticking with `var` here.
Truth is, there were no guidelines for this so there is plenty of mixed cases all over the place.

Reply:
Totally, `var` will leak out into the global `window` but yea, I'll switch all these back to `var` to remain consistent here in the codebase.